Package throws ignored error on npm install

## Case
Bug / Package Hygiene

## Issue
```
npm i rxdb@latest --save
```

Observe:

```
> rxdb@8.5.0 postinstall /Users/USER/PROJECT/node_modules/rxdb
> node scripts/postinstall.js || echo "ignore"

internal/modules/cjs/loader.js:584
    throw err;
    ^

Error: Cannot find module '/Users/USER/PROJECT/node_modules/rxdb/scripts/postinstall.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:754:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
ignore
```

The scripts directory is excluded from npm:

https://github.com/pubkey/rxdb/blob/master/.npmignore#L7

```
› ls ./rxdb/scripts/postinstall.js
gls: cannot access './rxdb/scripts/postinstall.js': No such file or directory
```

This issue does not appear to cause any problems, but it worries me seeing errors anywhere after npm install, especially ones like this, since I'm not sure if the package installed correctly or not.

## Info
  - Environment: Node.js
  - Adapter: N/A
  - Stack:  N/A

--- a/.npmignore
+++ b/.npmignore
@@ -4,7 +4,6 @@ examples/
 docs/
 docs-src/
 orga/
-scripts/
 test/
 test_tmp/
 *.js.map

